cmd/thv-operator/controllers/mcpserver_controller.go

```diff
@@ -1062,29 +1065,29 @@ func ensureRequiredEnvVars(env []corev1.EnvVar) []corev1.EnvVar {
 		}
 	}
 	if !xdgConfigHomeFound {
-		logger.Debugf("XDG_CONFIG_HOME not found, setting to /tmp")
+		ctxLogger.V(1).Info("XDG_CONFIG_HOME not found, setting to /tmp")
```

Comment:
I agree with using debug level for some traces, but do we have an option to enable it when we deploy either from helm charts or from the task commands?